app/konnect/analytics/dashboard.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Analytics dashboard
+---
+
+![service graph](/assets/images/products/konnect/analytics/konnect-summary-dashboard.png){:.image-border}
+
+
+You can access a summary dashboard by navigating to **Analytics** > **Summary**. The summary dashboard shows performance and health statistics of all your APIs across your organization on a single page.
+For greater insights into your service usage, access the dedicated {% konnect_icon analytics %} [Analytics](https://cloud.konghq.com/analytics) page.
+
+From {% konnect_icon analytics %} Analytics, you can view dashboards, access historical data for a range greater than 30 days, and customize the entities in a report:
```

Comment:
From here on, we only have the old content that is not correct anymore. Can we update this to just talk about what you can see today in the new dashboard?